README.md

```diff
@@ -1,9 +1,23 @@
 # Nava Labs Decision Support Tool
+
 Welcome! You are at the root of the Nava Labs Decision Support Tool pilot repo.
 
 ## Local Development
+
 To set up your local development environment, follow the instructions in [Getting Started](docs/app/getting-started.md).
 
+## Installing Ollama
+
+You can install ollama on macOSX by calling `brew install ollama`, for other operating systems or the desktop app see the [Ollama repository](https://github.com/ollama/ollama).
+
+To download a model run `ollama pull model_name`
+
+Ex: `ollama pull llama2:7b`
+
+To start Ollama without the desktop app run `ollama serve`
+
+To configure a local secret to enable Ollama add `OLLAMA_HOST=http://host.docker.internal:11434` to your `.env` file.
```

Comment:
Setting `OLLAMA_HOST` is not needed if running locally without Docker. Using `host.docker.internal` assumes you're running Docker, right?

Reply:
(I don't feel strongly about it, if folks do like having the app running locally, we can revise the approach in this ticket -- would need a way to communicate to `get_models` that Ollama is available as a local service, but that seems reasonable)

Reply:
> would need a way to communicate to get_models that Ollama is available as a local service

Using the default `http://localhost:11434` should work.

Reply:
@ccheng26 Reverting back to use `host.docker.internal` sounds like the simplest solution and aligns with the original intent in the ticket. [Related Slack thread](https://nava.slack.com/archives/C06DP498D1D/p1720473007216669?thread_ts=1720472024.809749&cid=C06DP498D1D)